CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log for hipBLAS
 
+## (Unreleased) hipBLAS 1.0.0
+### Changed
+- added const qualifier to pointers for the batched version of hipBLAS functions where missing
```

Comment:
Thanks @daineAMD for updating the CHANGELOG.md. 
As the const qualifier was also added to all variants of hipBLAS functions. It could be reworded as:- Added const qualifier to hipBLAS functions (swap, sbmv, spmv, symv, trsm) where missing.